data_filter_mongodb/README.md

```diff
@@ -0,0 +1,316 @@
+
+### Deploying in Kubernetes:
+
+Pre-requisites:
+
+1. kubectl
+2. kind - https://kind.sigs.k8s.io/
+
+#### Step 1:
+
+Create a Kubernetes cluster 
+
+```
+kind create cluster
+```
+
+#### Step 2:
+
+Running opa-mongo server with mongo database
+
+```
+kubectl create -f https://github.com/open-policy-agent/contrib/blob/master/data_filter_mongodb/k8s/opa-mongo.yaml
+```
+
+By now you have running opa-mongo server with mongo database. Now let's query the data from opa-mongo server.
+
+### View the data that exists in mongo database:
+
+You can view all the data that exist in the mongo database by accessing the below URL. This endpoint is exposed for viewing the data that exists in the database.
+
+```
+http://localhost:9095/records
+```
+
+This opa-mongo server is designed to retrieve employee details. The employee details can be seen either by the employee or by a respective manager. 
+
+Below are records that are inserted on opa server startup:
+
+```
+[
+  {
+    "Name": "john",
+    "Designation": "lead engineer",
+    "Salary": 270000,
+    "Email": "john@opa.com",
+    "Mobile": "1233743438738",
+    "Manager": "danerys"
+  },
+  {
+    "Name": "arya",
+    "Designation": "software engineer",
+    "Salary": 90000,
+    "Email": "arya@opa.com",
+    "Mobile": "1233746238738",
+    "Manager": "john"
+  },
+  {
+    "Name": "tyrian",
+    "Designation": "senior software engineer",
+    "Salary": 250000,
+    "Email": "tyrian@opa.com",
+    "Mobile": "123336238738",
+    "Manager": "danerys"
+  },
+  {
+    "Name": "jamie",
+    "Designation": "lead engineer",
+    "Salary": 70000,
+    "Email": "jamie@opa.com",
+    "Mobile": "1233746238738",
+    "Manager": "danerys"
+  },
+  {
+    "Name": "jeffrey",
+    "Designation": "software engineer",
+    "Salary": 60000,
+    "Email": "jeffrey@opa.com",
+    "Mobile": "1233746238738",
+    "Manager": "jamie"
+  },
+  {
+    "Name": "sansa",
+    "Designation": "senior software engineer",
+    "Salary": 80000,
+    "Email": "sansa@opa.com",
+    "Mobile": "1233746238738",
+    "Manager": "john"
+  },
+  {
+    "Name": "ramsay",
+    "Designation": "software engineer",
+    "Salary": 70000,
+    "Email": "ramsay@opa.com",
+    "Mobile": "1233746238738",
+    "Manager": "john"
+  },
+  {
+    "Name": "cersei",
+    "Designation": "senior software engineer",
+    "Salary": 170000,
+    "Email": "cersei@opa.com",
+    "Mobile": "1233746238738",
+    "Manager": "jamie"
+  },
+  {
+    "Name": "theon",
+    "Designation": "software engineer",
+    "Salary": 75000,
+    "Email": "theon@opa.com",
+    "Mobile": "1233746238738",
+    "Manager": "john"
+  },
+  {
+    "Name": "rob",
+    "Designation": "software engineer",
+    "Salary": 75000,
+    "Email": "rob@opa.com",
+    "Mobile": "1343238738",
+    "Manager": "john "
+  },
+  {
+    "Name": "danerys",
+    "Designation": "director of engineering",
+    "Salary": 350000,
+    "Email": "danerys@opa.com",
+    "Mobile": "12332423738",
+    "Manager": "nil"
```

Comment:
Nit: Why use `nil` here v/s not including the field itself.

Reply:
This is the response from `/records` API call, As the field is empty, the response is emitted as `nil` on API evocationl.